ferrilata-devilline/Models/DAOs/Badge.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ferrilata_devilline.Models.DAOs;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
```

Comment:
... ok, so using "git mv" did not work, and this still appears as a recreation instead of a modification. :( Sorry.